CDCgov/cdc-open-viz
# [DEV-8527] Resolve sparkline issue

### Test Steps

- Load the example sparkline file thats included on this ticket
- Under Filters > Filter > Uncheck the box that says show filter input
- Verify that the filter helper text is removed
